_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,205 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.4.9: # Images of Text (No Exception) (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+    - Times Square image released into the public domain by (WT-shared) Ypsilonatshared at wts wikivoyage.
+---
+
+## Applicability
+
+This rule applies to any [visible][] [embedded image][] that is not [marked as decorative][].
+
+## Expectation
+
+For each of the image resources referenced by the [image sources][] of the test target, at least one of the following is true:
+
+- the image resource does not contain text expressing anything in a [human language][]; or
+- the image resource contains text and the text is not the most significant visual content in the image; or
```

Comment:
I'm not sure about "most significant" as the criteria. If I have an arrow with the text "one way", is that text the most significant? Maybe better to say something like "not significant part of the visual content in the image" instead?

Reply:
Good suggestion.